formula/temporal_tables.rb

```diff
@@ -0,0 +1,37 @@
+class TemporalTables < Formula
+  desc "Temporal Tables PostgreSQL Extension"
+  homepage "http://pgxn.org/dist/temporal_tables/"
+  url "https://github.com/arkhipov/temporal_tables/archive/v1.1.0.tar.gz"
+  sha256 "1fe210a349d1418d097f229c36f30a1daef1ff17cf0f027685171c52e366308a"
+
+  depends_on :postgresql
```

Comment:
It's fine to use it it's just you were assuming the `postgresql` formula would be used.